plugins/github/client/Settings.tsx

```diff
@@ -0,0 +1,179 @@
+import format from "date-fns/format";
+import filter from "lodash/filter";
+import { observer } from "mobx-react";
+import * as React from "react";
+import { useTranslation, Trans } from "react-i18next";
+import { IntegrationService, IntegrationType } from "@shared/types";
+import Integration from "~/models/Integration";
+import Avatar from "~/components/Avatar";
+import { AvatarSize } from "~/components/Avatar/Avatar";
+import Button from "~/components/Button";
+import Heading from "~/components/Heading";
+import List from "~/components/List";
+import ListItem from "~/components/List/Item";
+import Notice from "~/components/Notice";
+import Scene from "~/components/Scene";
+import Text from "~/components/Text";
+import env from "~/env";
+import useCurrentTeam from "~/hooks/useCurrentTeam";
+import useQuery from "~/hooks/useQuery";
+import useStores from "~/hooks/useStores";
+import GitHubIcon from "./Icon";
+import GitHubButton from "./components/GitHubButton";
+
+function GitHub() {
+  const team = useCurrentTeam();
+  const { integrations } = useStores();
+  const { t } = useTranslation();
+  const query = useQuery();
+  const error = query.get("error");
+  const [relationsLoaded, setLoaded] = React.useState(false);
+
+  React.useEffect(() => {
+    void integrations.fetchPage({
+      service: IntegrationService.GitHub,
+      limit: 100,
```

Comment:
I know this is copied from the other screens, but you implemented `fetchAll` now so can use that here